modules/social_features/social_profile/modules/social_profile_preview/templates/profile--profile--preview.html.twig

```diff
@@ -54,5 +54,9 @@
     </div>
   </div>
 
-  {{ content|without('field_profile_image', 'uid', 'flag_follow_user', 'private_message_link') }}
+  {% if profile.field_profile_summary.isEmpty() %}
+    {{ content.field_profile_self_introduction }}
```

Comment:
@OH17L I think we can try to strip tags for `field_profile_self_introduction` since for now font style for `field_profile_self_introduction` and `field_profile_summary` look different because of tag `<p>` in `field_profile_self_introduction`.

### field_profile_summary
![610fccfab975db7bc7429a061dfa1578](https://user-images.githubusercontent.com/10220937/162801336-725ce85c-1414-4d6b-9c83-3d2f045c9d1f.png)

### field_profile_self_introduction
![0b1945d8132fa806f5ac727758b16326](https://user-images.githubusercontent.com/10220937/162801364-7726d793-116b-4c09-ba32-11cebf789dd7.png)

Reply:
@OH17L, please sub ticket for me and I'll add styles for both cases.

Reply:
@agami4, I'd say that it should be fixed on the back-end side so I'd do it.

Reply:
@Ressinel This will be fixed in #2846, but can we merge this PR first, since without this PR new Profile Summary field and empty Profile Preview don't make much sense in release 11.2?

Reply:
@ribel Yes, since the Social Profile Preview module is experimental I think we can omit this small FE issue.